D/D.sublime-syntax

```diff
@@ -8,339 +8,2057 @@ file_extensions:
+      set: value-after
+    - include: not-whitespace-illegal
+  value-bracket-after:
+    - match: '\]'
+      scope: punctuation.section.brackets.end.d
+      set: value-after
+    - include: not-whitespace-illegal
+
+  value-list:
+    - match: '(?=\)|}|]|;)'
+      pop: true
+    - match: '(?=\S)'
+      set: [value-list-after, value]
+  value-list-after:
+    - match: '(?=\)|}|]|;)'
+      pop: true
+    - match: ','
+      scope: punctuation.separator.sequence.d
+      set: value-list
+    - include: not-whitespace-illegal
+
+  value-array-list:
+    - match: '(?=\])'
+      pop: true
+    - match: '(?=\S)'
+      set: [value-array-list-after, value]
+  value-array-list-after:
+    - match: ':'
+      scope: punctuation.separator.mapping.key-value.d
+      set: [value-array-list-after-value, value]
+    - include: value-array-list-after-value
+  value-array-list-after-value:
+    - match: '(?=\])'
+      pop: true
+    - match: ','
+      scope: punctuation.separator.sequence.d
+      set: value-array-list
+    - include: not-whitespace-illegal
+
+  alias-in:
+    - match: '\b(alias)\b(?=[^;]*\bthis\s*;)'
+      scope: keyword.control.alias.d
+      push:  [alias-this-after, identifier]
+    - match: '\b(alias)\b'
+      scope: keyword.control.alias.d
       push:
-        - meta_scope: comment.block.d
-        - match: \*/
-          scope: punctuation.definition.comment.d
-          pop: true
-    - match: /\+
-      scope: punctuation.definition.comment.d
+        - match: '{{attribute_lookahead}}'
+          set: [alias-old, type]
+        - match: '\b({{name}})\s*(?=\(|=)'
+          captures:
+            1: entity.name.type.d
+          set: alias-new
+        - match: (?=\S)
+          set: [alias-old, type]
+  alias-this-after:
+    - match: \b(this)\b
+      scope: keyword.control.alias.d
+      set: expect-end-of-line
+    - include: not-whitespace-illegal-pop
+  alias-old:
+    - match: '\b({{name}})\b'
+      scope: entity.name.type.d
+      set:
+        - match: ','
+          scope: punctuation.separator.sequence.d
+          set: alias-old
+        - include: end-of-line
+        - include: not-whitespace-illegal
+    - include: not-whitespace-illegal
+  alias-new:
+    - match: '(?=\()'
+      push: function-argument-definition-list
+    - match: '(?==)'
+      set: alias-new-assign
+    - include: not-whitespace-illegal
+  alias-new-assign:
+    - match: '='
+      scope: keyword.operator.assignment.d
+      set: [alias-new-after-type, value]
+    - include: not-whitespace-illegal
+  alias-new-after-type:
+    - include: end-of-line
+    - match: ','
+      scope: punctuation.separator.sequence.d
+      set:
+        - match: '\b{{name}}\b'
+          scope: entity.name.type.d
+          set: alias-new
+        - include: not-whitespace-illegal
+    - include: not-whitespace-illegal
+
+  class-in:
+    - match: '\b(class)\b'
+      scope: storage.type.class.d keyword.declaration.class.d
```

Comment:
Ideally, everything from here to the closing brace of the class definition would get `meta.class`. However, at the least, from here until the opening brace should get `meta.class`.